relaybot.py

```diff
@@ -52,7 +53,10 @@ def get(option):
             options["nickServPassword"] = get("nickServPassword")
 
         factory = factory(options)
-        reactor.connectTCP(options['host'], int(options['port']), factory, int(options['timeout']))
+        if options['ssl']:

```

Comment:
That'd probably be better. `boolean(options['ssl'])` would still be specified-or-not - what's the pythonic way to turn a string into a boolean in "do what I mean" fashion?